docs/primer/index.md

```diff
@@ -1,7 +1,18 @@
 # Primer
 
+In his Primer, you will find resources that cover the following topics:
+
+- [Open Fibre Data](openfibredata) introduces open fibre data and helps to answer the question of why we might want to publish it.
+- [The Open Fibre Data Standard](openfibredatastandard) (OFDS) introduces the OFDS and OFDS data, and helps to answer the question of why we might want to use the OFDS.
+- [Scope and Key Concepts](scopeandkeyconcepts) introduces layers, actors and business models in fibre optic networks, and provides some definitions for key concepts in the OFDS.
+
+Looking to learn how to publish and use open fibre data? Read the [guidance](../guidance).
+
+Looking for details of the OFDS data model? Read the [reference](../reference).
+
```

Comment:
I've suggested a copy-edit based on the guidance and tools in the [readability](https://ofds-standard-development-handbook.readthedocs.io/en/latest/style/common_conventions.html#readability) section of the handbook (I find [Hemingway Editor](https://hemingwayapp.com/) very useful). In particular:

* Use clear and concise sentences for the benefit of non-native English speakers
* Use second person narrative (you) rather than first person narrative (we)
* Use consistent capitalisation
* Avoid incomplete sentences

```suggestion

This Primer provides an introduction to:

* [Open fibre data](openfibredata) and the reasons for publishing it.
* [The Open Fibre Data Standard](openfibredatastandard) (OFDS) and the reasons for using it.
* [The scope of OFDS and key concepts](scopeandkeyconcepts) that you need to understand

Do you want to learn how to publish or use open fibre data? Read the [guidance](../guidance).

Are you looking for details of the OFDS data model and the rules for publishers? Read the [reference](../reference).
```